godel_plugins/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ find_package(catkin REQUIRED COMPONENTS
 )
 
 # c++11
-ADD_DEFINITIONS("-std=c++0x")
+#ADD_DEFINITIONS("-std=c++0x")
```

Comment:
While I don't think it needs to be eliminated, it's not necessarily needed either.  Would you prefer I delete the flag all-together or switch it back on?